pages/learn/glossary.md

```diff
@@ -137,6 +141,14 @@ The following are considered tools:
 
 Applications which use JSON Schema internally without exposing that functionality in some way, for example, validating configuration files or web requests, are not considered tooling.
 
+### validation result
+
+The validation result in the context of JSON Schema refers to the outcome of applying the entire JSON Schema to the entire instance document. This application should yield a boolean assertion, indicating whether the instance document conforms to the rules and constraints specified in the schema. The validation result signifies whether the instance document passes or fails validation against the [schema](#schema).
```

Comment:
A validation result can be much more than a boolean assertion - there's a whole set of output formats that can be the result of validation, of which a boolean is the simplest.

Reply:
I think this is fine for "validation" result.  I'd consider the rest of the output to be "evaluation" results.  Not sure if we need to split that particular hair here, though.

Reply:
When I use the term "validation result", I always mean just the boolean result. I wouldn't be against using an expanded definition here, but I've never used it that way in the past.